_data/simple-icons.json

```diff
@@ -15182,6 +15182,11 @@
             "source": "https://sites.google.com/sumologic.com/sumo-logic-brand/home",
             "guidelines": "https://sites.google.com/sumologic.com/sumo-logic-brand/home"
         },
+        {
+            "title": "Suno",
+            "hex": "FFFFFF",
```

Comment:
My only question is I can't figure out / decide if we should be using `#000000` here or `#ffffff`... 🤔

Reply:
I'm also finding it challenging to decide. On their website, in some areas, they use a white logo within a black rounded square, while the header uses a black logo. But changing to `#000000` is not a problem I think.

Reply:
I think let's change to `#000000`:
![suno](https://github.com/simple-icons/simple-icons/assets/47499684/ac746f0a-ff9d-469d-a06b-68503eba0633)

Reply:
@uncenter updated